src/main.c

```diff
@@ -314,6 +315,9 @@ parse_arg_list(int argc, char **argv, char ***fuse_argv, int *fuse_argc)
                 CONFIG.http_headers =
                     curl_slist_append(CONFIG.http_headers, strdup(optarg));
                 break;
+            case 27:
+                CONFIG.cache_clear = 1;
```

Comment:
You should just call a function, clear the cache folders, and exit. There is no point of letting other functionalities of the program to run.

Reply:
https://github.com/fangfufu/httpdirfs/compare/f0bb59e56b619995593f1ffc36f69bf2f0f39340..9946e0a823887b7e1b49d98e84c5be8177bc7e55

Reply:
For context, the PR clears the cache associated with a URL, not the entire cache folder for httpdirfs. Are you suggesting the clear option should wipe the cache for any previous URL the user has visited?

Reply:
The original issue says:

> Greetings, I'm using the cache function and I understand it is permanent, but I need to clear it as it got too big, restarting is not clearing it. what is the cleanest way to clear it? I have around 1300 mounted directories, so what should I do? turn the mount off and delete the path content?

The user wanted their 1300 mounted directories to be addressed. If you read my reply, I gave them instruction on how to clear them. This pull request does not address the original issue.

Reply:
Hmmm...thank you for the perspective. I've updated the commit with a better understanding.

https://github.com/fangfufu/httpdirfs/commit/16f83920da5c365aae4b4747a1b4fea30af274ea